content/en/Integrations/IntegrationBuilder/How to Guides/add-pentest-title-to-jira-issue-summary.md

```diff
@@ -0,0 +1,80 @@
+Jira issues created by their recipes to contain the title of the relevant pentest.
+Please follow the [Create Jira tickets for Findings guide](/integrations/integrationbuilder/how-to-guides/jira-cloud-migration) before proceeding with this guide.
+
+{{% /pageinfo %}}
+
+1. Navigate to the __[Cobalt > Jira Cloud] Push pentest finding to Jira__ recipe that you set up while [following the previous guide](/integrations/integrationbuilder/how-to-guides/jira-cloud-migration).
+
+2. Open the edit view for the recipe by clicking the __Edit__ button.
+
+3. Hover your cursor above the __Monitor__ block that encapsulates the __Create issue in Jira__ action. An __Add step__ button will appear. Click on it.
+
+![add step](/integrations/integration_builder/how_to_guides/add-pentest-title-to-jira-issue-summary/1_add_step.png)
+
+4. Choose the __Handle errors__ option.
+
+![choose handle errors](/integrations/integration_builder/how_to_guides/add-pentest-title-to-jira-issue-summary/2_choose_handle_errors.png)
+
+5. Click on the empty action slot nested within the __Monitor__ block. A configuration pane will open on the right. Click on the __Cobalt Connector__.
+
+![add cobalt connector](/integrations/integration_builder/how_to_guides/add-pentest-title-to-jira-issue-summary/3_add_cobalt_connector_action.png)
+
+6. Choose the __Get pentest__ action.
+
+![get pentest action](/integrations/integration_builder/how_to_guides/add-pentest-title-to-jira-issue-summary/4_choose_get_pentest_action.png)
+
+7. You now need to map the ID of the pentest associated to the finding into the __Pentest ID__ input field.
+   Once you click into the __Pentest ID__ input field, the __Recipe data__ drawer will open.
+   Within the __Recipe data__ drawer, look for the __Pentest finding state updated__ trigger and expand it to see all of its outputs.
+   Find the __Pentest ID__ output and click on it to map it to the __Pentest ID__ input field.
+
+![map in pentest id](/integrations/integration_builder/how_to_guides/add-pentest-title-to-jira-issue-summary/5_map_in_pentest_id.png)
+
+![how it should look](/integrations/integration_builder/how_to_guides/add-pentest-title-to-jira-issue-summary/6_how_it_should_look.png)
+
+8. Click on the __Retry__ action that is below the __Get pentest__ action. Configure it to retry 3 times and to wait 5 seconds between retries.
+   This gives the recipe a decent chance to automatically recover from any network failures.
+
+![configure retry](/integrations/integration_builder/how_to_guides/add-pentest-title-to-jira-issue-summary/7_retry_get_pentest_3_times.png)
+
+9. Below the __Retry__ action, there is an arrow labeled __if error persists__. Below this arrow there is an empty action slot. 
+   Hover over it and click the elipsis that appears to the right of the action slot.
+   In the menu that appears, click __Delete__ to remove the action slot.
```

Comment:
I could not find a way to avoid this either 😕

Reply:
😞